Build/GameSettings.h

```diff
@@ -73,11 +73,10 @@ struct GAME_SETTINGS
 //Enums for the difficulty levels
 enum
 {
-	DIF_LEVEL_ZERO,
-	DIF_LEVEL_EASY,
-	DIF_LEVEL_MEDIUM,
-	DIF_LEVEL_HARD,
-	DIF_LEVEL_FOUR,
+	DIF_LEVEL_EASY = 1,
```

Comment:
Where does this assignment happen, because there seems to be no change for that and before both enums were `0`-based.